.jvmopts

```diff
@@ -1,4 +1,5 @@
 -Dfile.encoding=UTF8
+-Djava.security.egd=file:/dev/./urandom
```

Comment:
@raboof have you checked that this parameter is not effectively inherited in the forked process?